src/media/js/previews-buttons.js

```diff
@@ -0,0 +1,59 @@
+/*
+    Arrow buttons for navigating back and forth in a preview tray that tell
+    Flipsnap what to do.
+
+    - Buttons disabled if there is no room to scroll left or right.
+    - We limit Flipsnap's max scroll position by 1 so we cant't scroll too far.
+    - If no room to scroll left or right (e.g., not enough previews), hide
+      buttons completely.
+*/
+define('previews-buttons', ['core/z'], function(z) {
+    function attach(slider, $container, opts) {
+        var isLightbox = opts && opts.isLightbox;
+
+        $container.find('.arrow-btn-prev, .arrow-btn-next').remove();
+
+        var $prevBtn = $('<a class="arrow-btn arrow-btn-prev"></a>');
+        var $nextBtn = $('<a class="arrow-btn arrow-btn-next"></a>');
+        var $previewBtns = $prevBtn.add($nextBtn);
+
+        function setPreviewBtnState() {
+            // Toggles whether buttons are hidden or disabled.
+            $prevBtn.add($nextBtn).addClass('arrow-btn-disabled');
+
+            var hasButton = false;
+            if (slider.hasNext() &&
+                slider.currentPoint !== slider._maxPoint - 1) {
```

Comment:
How would it decrease in width? I found this works for X number of previews given our current preview trays.